__tests__/search.js

```diff
@@ -35,28 +35,53 @@ class MockVFSServiceProvider extends ServiceProvider {
     ];
   }
 }
+class MySearchAdapter {
+    constructor(core) {
+        this.core = core;
 
+    }
+
+    destroy() {
+    }
+
+    async init() {
+    }
+
+    async search(pattern) {
+        const list = [
+            {
+             path: 'jest:/test.jpg',
+             filename: pattern+'.jpg'   
+            }
+        ];
+        return list;
+    }
+}
 describe('Search', () => {
   let core;
   let search;
-
+  let options = {};
```

Comment:
Just place options directly into the `Search`  constructor. This is only tested once anyways.

Reply:
By "place it directly into the constructor", I meant to remove this entirely.